<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Orbiter Addons Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/icon/satellite.ico</normaloff>:/res/icon/satellite.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Orbiter Addons Manager</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="addonsList">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="statusTip">
       <string>Installed addons</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonsLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="statusTip">
         <string>Install/Add an addon to database</string>
        </property>
        <property name="text">
         <string>Install/Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="uninstallButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="statusTip">
         <string>Uninstall the selected addon</string>
        </property>
        <property name="text">
         <string>Uninstall</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPath"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPath">
   <property name="text">
    <string>Set Orbiter path</string>
   </property>
   <property name="statusTip">
    <string>Set Orbiter installation directory</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About Orbiter Addons Manager</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
